src/main/java/uk/gov/hmcts/reform/sscs/service/EvidenceManagementService.java

```diff
@@ -33,10 +40,22 @@ public UploadResponse upload(List<MultipartFile> files) {
 
         try {
             return documentUploadClientApi
-                    .upload(DUMMY_OAUTH_2_TOKEN, serviceAuthorization, files);
+                    .upload(S2S_TOKEN, serviceAuthorization, files);
         } catch (HttpClientErrorException httpClientErrorException) {
             throw new UnsupportedDocumentTypeException(httpClientErrorException);
         }
     }
 
+    public byte[] download(String documentId) {
+        String serviceAuthorization = authTokenGenerator.generate();
+
+        try {
+            ResponseEntity<Resource> responseEntity =  documentDownloadClientApi.downloadBinary(S2S_TOKEN, serviceAuthorization, documentId);
+
+            ByteArrayResource resource = (ByteArrayResource) responseEntity.getBody();
+            return resource.getByteArray();
+        } catch (HttpClientErrorException httpClientErrorException) {
+            throw new UnsupportedDocumentTypeException(httpClientErrorException);
+        }
```

Comment:
Log the exception before throwing back to the client. We had an issue on friday and we were not able correlate the failure from SYA to TCA API.

Reply:
Will do, thanks.

Reply:
Done